screens/Issuers/IssuersScreen.tsx

```diff
@@ -124,24 +129,56 @@ export const IssuersScreen: React.FC<
     );
   }
 
+  const showAllIssuers = () => {
```

Comment:
No need for separate method showallIssuers. we can handle it in the search issuers itself by passing empty text.

Reply:
I tried, but it wouldn't work to show the list of items when entered into the "**Add new card**" screen.
So followed with onLayout props in TextInput with the above function (**showAllIssuers**)